src/components/thumbnail/thumbnail.jsx

```diff
@@ -32,7 +33,10 @@ var Thumbnail = React.createClass({
         if (this.props.creator) {
             extra.push(
                 <div key="creator" className="thumbnail-creator">
-                    by <a href={'/users/' + this.props.creator + '/'}>{this.props.creator}</a>
+                    <FormattedMessage id={'thumbnail.by'} />{' '}
+                    <a href={'/users/' + this.props.creator + '/'}>
+                        {this.props.creator}
+                    </a>
```

Comment:
This is a good example of why we need to upgrade our localization tool — just translating "by" is going to be ambiguous in a lot of languages, and may need the context of the creator username to be accurate.